packages/paste-website/src/pages/experiences/console/alarm-setup/index.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: About Paste
+description: An overview of what Paste is, is not, and how to use the platform.
+slug: /introduction/about-paste
+---
+
+import {graphql} from 'gatsby';
+
+export const pageQuery = graphql`
+  {
+    mdx(frontmatter: {slug: {eq: "/introduction/about-paste/"}}) {
```

Comment:
question: seems like an inaccurate link and frontmatter (line 1-4) for this page. Did you mean to commit this page?

Reply:
Ah, you're right, this info is inaccurate! I've corrected it and also just added the empty state pattern here, should be all good now!